core/src/main/scala/libretto/ClosedDSL.scala

```diff
@@ -79,6 +79,7 @@ trait ClosedDSL extends CoreDSL {
       LambdaContext,
     ): $[B]
 
+    // FixMe: Changing this to a `extension` will crash the compiler. To investigate further…
```

Comment:
Well, the comments says it all…

I guess I would need help to minimize this issues so it can be brought into attention of the Scala compiler team. Scala 3.3.1 is about to come out, I guess we should hurry to get a fix in.

Reply:
Let's try to get the easy ones in in this PR. I didn't think eliminating implicits would be done in one go. You have done exceedingly well in this regard :)

Let's resolve it as part of a separate issue. Even if we manage to minimize the compiler problem, I wouldn't hold my breath with getting a fix in 3.3.1 (unless it's a regression or very easy for someone to fix).